lib/rust/frp/src/io/keyboard.rs

```diff
@@ -440,17 +439,14 @@ pub struct DomBindings {
 
 impl DomBindings {
     /// Create new Keyboard and Frp bindings.
-    pub fn new(keyboard: &Keyboard, current_event: &JsEvent) -> Self {
-        let key_down = Listener::new_key_down(current_event.handler(
+    pub fn new(keyboard: &Keyboard) -> Self {
+        let key_down = Listener::new_key_down(
             f!((event:&KeyboardEvent) keyboard.source.down.emit(KeyWithCode::from(event))),
-        ));
-        let key_up =
-            Listener::new_key_up(current_event.handler(
-                f!((event:&KeyboardEvent) keyboard.source.up.emit(KeyWithCode::from(event))),
-            ));
-        let blur = Listener::new_blur(
-            current_event.handler(f_!(keyboard.source.window_defocused.emit(()))),
         );
+        let key_up = Listener::new_key_up(
+            f!((event:&KeyboardEvent) keyboard.source.up.emit(KeyWithCode::from(event))),
+        );
+        let blur = Listener::new_blur(f_!(keyboard.source.window_defocused.emit(())));
```

Comment:
Why do we not set current_event anymore?

Reply:
As agreed via phone, it's OK as the JsEvent was removed.